R/drive_upload.R

```diff
@@ -40,11 +41,33 @@
 #'
 #' ## clean-up
 #' drive_find("BioC_mirrors") %>% drive_rm()
+#'
+#' ## Upload a file with multiple parents
+#'
+#' ## Create mutltiple parent directories
+#' folder_1 <- drive_mkdir("folder_one")
+#' folder_2 <- drive_mkdir("folder_two")
+#'
+#' ## Upload the file into multiple parents. Notice here we are using
+#' ## the `addParents` parameter via the `...`.
+#' logo <- drive_upload(
+#'   R.home('doc/html/logo.jpg'),
+#'   folder_1,
+#'   addParents = folder_2$id
```

Comment:
Now that I've read the tests, I understand this better now. Your approach prevents root folder from being a parent, IIUC.